plugins/powerd/fu-plugin-powerd.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2021 Twain Byrnes <binarynewts@google.com>
+ * Copyright (C) 2021 George Popoola <gpopoola@google.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <fwupdplugin.h>
+
+struct FuPluginData {
+	GDBusProxy *proxy; /* nullable */
+};
+
+void
+fu_plugin_init (FuPlugin *plugin)
+{
+	fu_plugin_set_build_hash (plugin, FU_BUILD_HASH);
+}
+
+gboolean
+fu_plugin_startup (FuPlugin *plugin, GError **error)
+{
+	g_set_error (
+	    error, G_IO_ERROR, G_IO_ERROR_NOT_SUPPORTED, "not supported yet");
+	return TRUE;
```

Comment:
I thought we had suggested this to return FALSE in the skeleton?

Reply:
Right, sorry about that, it got changed while I was resolving a merge conflict.